storage/object/how-to/create-bucket-policy.mdx

```diff
@@ -0,0 +1,237 @@
+---
+meta:
+  title: How to create and manage bucket policies using the console
+  description: This page shows how to create and manage bucket policies using the Scaleway console
+content:
+  h1: How to create and manage bucket policies using the console
+  paragraph: This page shows how to create and manage bucket policies using the Scaleway console
+tags: bucket policy bucket console object storage s3 access
+dates:
+  validation: 2024-05-30
+  posted: 2024-05-30
+categories:
+  - storage
+  - object-storage
+---
+
+Scaleway Object Storage allows you to create [bucket policies](/storage/object/concepts/#bucket-policy) directly from the [Scaleway console](https://console.scaleway.com). 
+
+To create and apply a bucket policy from the AWS CLI, refer to the [dedicated documentation](/storage/object/api-cli/create-bucket-policy/).
+
+<Macro id="storage-bucket-policy" />
+
+<Macro id="requirements" />
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- [Created a bucket](/storage/object/how-to/create-a-bucket/)
+
+## How to create a bucket policy
+
+1. Click **Object Storage** on the left side menu of the console. The Object Storage dashboard displays.
+
+2. Click the **Bucket policies** tab, then click **+ Create bucket policy**. A creation form displays.
+
+3. Enter a name for your bucket policy.
+
+4. Select the bucket to which you want to apply the policy.
+
+    <Message type="note">
+    - You can only apply a policy to a single bucket, but you can [duplicate an existing policy](/storage/object/how-to/create-bucket-policy/#how-to-apply-an-existing-policy-to-another-bucket) and apply it to another bucket.
+    - If you change the target bucket of an existing policy, the bucket policy will be removed from the original bucket, and applied to the selected bucket.
+    </Message>
+
+5. Select your preferred creation method:
+
+    - The **policy generator** to add [statements](/storage/object/api-cli/bucket-policy/#statement) using a form.
+    
+    - The **JSON editor** to manually write or paste your bucket policy in a code editor.
+
+    <Message type="note">
+    Refer to the dedicated documentation on [bucket policies overview](/storage/object/api-cli/bucket-policy/) and [combining IAM and bucket policies](/storage/object/api-cli/combining-iam-and-object-storage/) for more information on how to build a bucket policy.
+    </Message>
+
+6. By default, the bucket policy contains a **secure statement** that allows every action to the user creating the policy. Keep the **Maintain access to bucket** box ticked to prevent accidental lockout from your bucket.
```

Comment:
```suggestion
6.  Keep the **Maintain access to bucket** box ticked to prevent accidental lockout from your bucket. By default, the bucket policy contains a **secure statement** that allows every action to the user creating the policy.
```